pyvista/plotting/renderer.py

```diff
@@ -1249,6 +1255,9 @@ def show_bounds(
         # set bounds
         if bounds is None:
             bounds = np.array(mesh.GetBounds())
+        else:
+            bounds = np.array(bounds, dtype=np.float_)
+
         if isinstance(padding, (int, float)) and 0.0 <= padding < 1.0:
             if not np.any(np.abs(bounds) == np.inf):
```

Comment:
I have a mind to change this too... for another PR. Mostly a note to self. This whole block could be made more concise.